dependabot/update_lag_status.py

```diff
@@ -107,56 +119,64 @@ def get_lag_info(module_name):
             timestamp_string = current_version.split("-")[2:4]
             timestamp = create_timestamp(timestamp_string[0], timestamp_string[1])
 
-    lang_version = ballerina_lang_version.split("-")
-    ballerina_timestamp = create_timestamp(lang_version[2], lang_version[3])
     update_timestamp = ballerina_timestamp - timestamp
-    delta = format_lag(update_timestamp)
-    days = str(delta)
+    days, hrs = format_lag(update_timestamp)
 
-    if delta == 0:
-        color = "brightgreen"
-    elif delta < 2:
-        color = "yellow"
+    color = get_lag_color(days)
+
+    return days, hrs, color
+
+
+def get_lag_button(module):
+    global updated_modules
+    days, hrs, color = get_lag_info(module[MODULE_NAME])
+    if days > 0:
+        lag_status = str(days) + "%20days"
+    elif hrs > 0:
+        lag_status = str(hrs) + "%20h"
     else:
-        color = "red"
+        lag_status = "no%20lag"
```

Comment:
Shall we say it as latest? or simply N/A? Else the lag is repeating..

Reply:
Sure I'll set it to N/A then